### PR TITLE
ADD: lightning address send support (closes #3690)

### LINKED ISSUE #3690: Lightning-address pay
https://github.com/BlueWallet/LndHub/issues/277

https://github.com/fiatjaf/lnurl-rfc/blob/luds/16.md

https://github.com/andrerfneves/lightning-address/blob/master/README.md

### PR DESCRIPTION
**Pull request by Overtorment**

### CODE REVIEW

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/f95hct90u728vjmbav5g73xaqw?device=pixel4

[download apk](https://lambda-download-android-build.herokuapp.com/download/8061)

File: screen/lnd/lnurlPay.js [hunk 1]
```diff
@@ -56,11 +56,16 @@ const LnurlPay = () => {
...
+        .then(setPayload)
+        .catch(error => {
+          alert(error.message);
+          pop();
+        });
       setLN(ln);
       setIsLoading(false);
     }
-  }, [lnurl]);
+  }, [lnurl, pop]);
```
Comment (marcosrdz) [inline screen/lnd/lnurlPay.js:68]: Don't think pop needs to be in useEffect dependencies as its not something we need to keep track of. what do u think @limpbrains  ?

**Review by marcosrdz — CHANGES_REQUESTED** [review]

File: screen/lnd/lnurlPay.js (hunk 1 above)
Comment (Overtorment) [reply screen/lnd/lnurlPay.js:68]: eslint complained that its not on deps list, so I added.
don't really understand whats going on here

Comment (limpbrains) [reply screen/lnd/lnurlPay.js:68]: I think it is better to follow eslint `react-hooks/exhaustive-deps` rule and list all deps in array, like it is here. 
Especially if you don't know whats going on :)
I hope react-navigation doesn't mutate pop() too often

**Review by limpbrains — APPROVED** [review]
According to code `asdasd @domain` is a valid address. I guess it's ok.

**Comment by Overtorment** [comment]
Good catch. Will patch it later

**Review by marcosrdz — APPROVED** [review]

**Comment by GladosBlueWallet** [comment]
Unbelievable. You, [subject name here], must be the pride of [subject hometown here]!

**Comment by xanoni** [comment]
nice, thank you

**Pull request merged**